features/app-manifest.md

```diff
@@ -8,6 +8,7 @@ spec_url: https://w3c.github.io/manifest/
 spec_repo: https://github.com/w3c/manifest
 chrome_ref: 6488656873259008
 ie_ref: Web Application Manifest
+opera_ref: 
```

Comment:
The empty values shouldn't be needed, but they don't hurt, so no need to remove them